pkg/utils/cached_env.go

```diff
@@ -0,0 +1,59 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package utils
```

Comment:
I kinda hate that we have this package.  We should open an issue to distribute this stuff to better named subpackages :(

Reply:
e.g. we could put this file into `package cacheenv` and have `cacheenv.Get()`.

Reply:
We will have an explosion of folders with single files. I am ok doing that, but perhaps something a little less specific, like ioutil? Alternatively, I can pull this into queue and keep it there since it is the only place this is used now.